device/core/test/_module_client_test.js

```diff
@@ -237,13 +246,13 @@ describe('ModuleClient', function () {
     });
 
     // Tests_SRS_NODE_MODULE_CLIENT_13_030: [ The value for the environment variable IOTEDGE_AUTHSCHEME must be sasToken. ]
-    describe('check IOTEDGE_AUTHSCHEME', function() {
+    describe('check IOTEDGE_AUTHSCHEME', function () {
```

Comment:
>  [](start = 49, length = 1)

nit: This is fairly irritating.

Reply:
Done.

---
In reply to: [222777498](https://github.com/Azure/azure-iot-sdk-node/pull/372#discussion_r222777498) [](ancestors = 222777498)